src/App.css

```diff
@@ -87,6 +87,20 @@ html,
   color: white;
 }
 
+.close-icon {
+  margin-left: 34rem;
```

Comment:
I think it would be better to position this icon with `position: absolute; right: somenumber, top: somenumber;`,  that `margin-left` can be harder to maintain if for any reason that dialog changes size

Reply:
![image](https://user-images.githubusercontent.com/80984726/147787405-80855d27-0787-431b-b0c1-3dfa764aad88.png)
This is my approach for this one, it works fine, sounds good?

Reply:
I added some comments to refactor the css and the classes